examples/qbv-example/build.gradle

```diff
@@ -0,0 +1,63 @@
+buildscript {
```

Comment:
So here's a tricky thing with these example projects - ideally, we want simple build.gradle files that use the much simpler syntax of:

```
plugins {
	id "com.marklogic.ml-gradle" version "4.6.0"
}
```

And most of the time in the real world, that's all that a user needs. 

But in order to create/test these, we need to use the more verbose buildscript syntax that you have here so we can depend on a local snapshot. 

Since we have `local-testing-project` for local testing, which also shows the correct buildscript syntax, change this over to use the simple syntax of a single `plugins` block. If I want to test the project locally, I'll just swap out `plugins` temporarily for the `buildscript` stuff.